target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -154,6 +154,10 @@ case "$FIRMWARE" in
 		caldata_extract "0:ART" 0x1000 0x2f20
 		ath10k_patch_mac $(mtd_get_mac_binary "0:MANUDATA" 0x6)
 		;;
+	teltonika,rutx10)
+		caldata_extract "0:ART" 0x1000 0x2f20
+		ath10k_patch_mac $(macaddr_add $(get_mac_label) 2)
```

Comment:
Please don't use get_mac_label here, it will result in undefined behavior.

Either read from flash or from eth0 if not possible otherwise.